src/content/patterns/creation-flows/creation-flows.mdx

```diff
@@ -0,0 +1,132 @@
+
+</AnchorLinks>
+
+### Modal
+
+If there are only one or two decisions to be made during a creation, use a small, standard modal overlaying the content where the creation was initiated. Avoid scrolling inside a small modal. 
+
+<ImageComponent caption="Example of a standard creation modal" cols="8">
+
+![Example of a standard creation modal](images/2.png)
+
+</ImageComponent>
+
+### Side panel
+
+If you have limited screen real estate, consider using a side panel. Side panels also keep the user aware of the context in which they were working. You may choose to use either the slide in or slide over panels.
+
+<ImageComponent caption="Example of a side panel creation flow" cols="8">
+
+![Example of a side panel creation flow](images/3.png)
+
+</ImageComponent>
+
+### Full page
+
+If your creation flow consists of multiple fields and/or supporting imagery, consider the full page option. Full page creation flows allow you to take advantage of maximum screen real estate.
+
+<ImageComponent caption="Example of a fill page creation flow" cols="8">
+
+![Example of a fill page creation flow](images/7.png)
+
+</ImageComponent>
+
+#### Supporting image
+
+To include a supporting image, place it on the right hand side of the window. 
+
+#### Advanced option
+
+The advanced option is used to avoid overwhelming users, while allowing advanced users to access all options. It can be applied to the side panel and the full page dialog flows.
+
+<ImageComponent caption="Example of advanced option" cols="4">
+
+![Example of advanced option](images/8.png)
+
+</ImageComponent>
+
+Have a minimal set of options visible by default, with the ability for the user to chose if they want to expose more ‘advanced’ options. If the user switches to advanced mode, remember this next time they perform the same task.
+
+## Multistep creation
+
+If the creation task is complex, consider breaking the process up into multiple steps. Display related or dependant options together. 
+
+Multistep creation follows the same principles as Standard creation, with the addition of multisteps.
+
+
+<AnchorLinks small>
+
+- [Side panel](#side-panel)
+- [Full page](#full-page)
+
+</AnchorLinks>
+
+### Side panel
+
+<ImageComponent caption="Example of a multi-step creation flow" cols="8">
+
+![Example of a multi-step creation flow](images/10.png)
+
+</ImageComponent>
+
+### Full page
+
+<ImageComponent caption="Start of multistep creation flow" cols="8">
+
+![Start of multistep creation flow](images/11.1.png)
+
+</ImageComponent>
+
+<ImageComponent caption="Multistep creation flow with image" cols="8">
+
+![Multistep creation flow with image](images/11.2.png)
+
+</ImageComponent>
+
+### Confirmation
+
+When any create action is a success, a notification banner can be presented to visually confirm this. 
+
+<ImageComponent caption="Example of a successful creation confirmation" cols="8">
+
+![Example of a successful creation confirmation"](images/12.png)
+
+</ImageComponent>
+
+#### Maintainers:
```

Comment:
Can we move the maintainer's section to the top of the page, afraid it will be missed at the bottom.

Reply:
Updated!